libs/local_lcos/tests/unit/CMakeLists.txt

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+set(tests
+  apply_local
+  apply_local_executor
+  async_local
+  async_local_executor
```

Comment:
The `apply` and `async` tests can stay I think. They'll go in a separate module.